libs/domains/cluster-metrics/feature/src/lib/cluster-card-node-usage/cluster-card-node-usage.tsx

```diff
@@ -0,0 +1,125 @@
+import { match } from 'ts-pattern'
+import { useCluster, useClusterRunningStatus } from '@qovery/domains/clusters/feature'
+import { CLUSTER_SETTINGS_RESOURCES_URL, CLUSTER_SETTINGS_URL, CLUSTER_URL } from '@qovery/shared/routes'
+import { Icon, Link, ProgressBar, Skeleton, Tooltip } from '@qovery/shared/ui'
+
+export interface ClusterCardNodeUsageProps {
+  organizationId: string
+  clusterId: string
+}
+
+export function ClusterCardNodeUsage({ organizationId, clusterId }: ClusterCardNodeUsageProps) {
+  const { data: runningStatus } = useClusterRunningStatus({
+    organizationId: organizationId,
+    clusterId: clusterId,
+  })
+
+  const runningStatusNotAvailable = typeof runningStatus !== 'object'
+
+  const { data: cluster } = useCluster({ organizationId, clusterId })
+
+  const shouldDisplayMinMaxNodes = match(cluster)
+    .with({ cloud_provider: 'GCP' }, () => false)
+    .with({ cloud_provider: 'ON_PREMISE' }, () => false)
+    .with({ cloud_provider: 'AWS', instance_type: 'KARPENTER' }, () => false)
+    .otherwise(() => true)
+
+  const totalNodes = (!shouldDisplayMinMaxNodes ? runningStatus?.nodes?.length : cluster?.max_running_nodes) || 0
+
+  const healthyNodes =
+    runningStatus?.nodes?.filter(
+      (node) => node.conditions?.find((condition) => condition.type === 'Ready')?.status === 'True'
+    ).length || 0
+
+  const warningNodes = Object.keys(runningStatus?.computed_status?.node_warnings || {}).length || 0
+
+  const deployingNodes =
+    runningStatus?.nodes?.filter(
+      (node) => node.conditions?.find((condition) => condition.type === 'Ready')?.status === 'False'
+    ).length || 0
+
+  const healthyPercentage = totalNodes > 0 ? (healthyNodes / totalNodes) * 100 : 0
+  const warningPercentage = totalNodes > 0 ? (warningNodes / totalNodes) * 100 : 0
+  const deployingPercentage = totalNodes > 0 ? (deployingNodes / totalNodes) * 100 : 0
```

Comment:
@RemiBonnet nitpicking: could be nice to create a helper to calculate the percentages and use it everywhere. What do you think?